layouts/kind/talk.html

```diff
@@ -0,0 +1,3 @@
+<h3>Talk abstract</h3>
+<%= yield %>
+<a class="btn btn-default" href="/2016-berlin/schedule/">Back to schedule</a>
```

Comment:
I'd go live with the 2017 page without speaker and talks, given we won't have any in the beginning? It'll probably be a  new task to create these pages once the CfP is over..